README.md

```diff
@@ -941,6 +941,7 @@ A curated list of amazingly awesome Elixir libraries, resources, and shiny thing
 * [mysqlex](https://github.com/tjheeta/mysqlex) - An Ecto-compatible wrapper around the mysql-otp library.
 * [neo4j_sips](https://github.com/florinpatrascu/neo4j_sips) - Neo4j driver for Elixir.
 * [neo4j_sips_models](https://github.com/florinpatrascu/neo4j_sips_models) - Minimalistic Model support for the Neo4j.Sips Elixir driver.
+* [paper_trail](https://hex.pm/packages/paper_trail) - Ecto plugin for tracking and recording all the changes in your database.
```

Comment:
Thanks for the contribution @izelnakri 
We use only github links, please replace the link...